app/views/list/show.html.slim

```diff
@@ -0,0 +1,34 @@
+.container-fluid
+  .row
+    .col-xs-12
+      .wishlist-header
+        | Requests
+      .wishlist-page
+        - if @requests.any?
+          .table-responsive
+            table#catalog-table.wishlist-table.table.table-hover
+              thead
+                tr
+                  th
+                  th
+                  th
+                    | Recipient
+                  th
+                    | Description
+                  th
+                    | Link
+                  th
+                    | Age
+                  th
+                    | Gender
+              tbody
+                = render partial: 'list/request_row_template', collection: @requests, as: :request
+        - else
+          h1 = "No requests have been created for this wishlist!"
+        
+        .wishlist-remove
+          button.btn.btn-danger[type="submit"]
+            | Remove Selected Requests
+        .wishlist-add
+          button.btn.btn-default[type="submit"]
```

Comment:
Yeah you're right - it should be `new_gift_request_path`. You don't need to flesh that out on the other end quite yet, but it's good to have it as a link rather than as a submit button (which would submit the above form, which you definitely don't want).